tests/Http/SignupTest.php

```diff
@@ -642,46 +642,6 @@ public function testSignupsIndexAsAdminWithFilters()
         $this->assertEquals(5, $decodedResponse['meta']['cursor']['count']);
     }
 
-    /**
-     * Test for retrieving all signups as admin filtering by quantity (in ascending or descending order).
-     *
-     * GET /api/v3/signups?orderBy=quantity,desc
-     * GET /api/v3/signups?orderBy=quantity,asc
-     *
-     * @return void
-     */
-    public function testSignupsIndexAsAdminWithOrderByQuantityFilter()
-    {
-        // Create 5 signups with different quantities
-        $signups = factory(Signup::class, 5)->create();
-        $quantity = 1;
-
-        foreach ($signups as $signup) {
-            $signup->quantity = $quantity++;
-            $signup->save();
-        }
-
-        // Order results by descending quantity
-        $response = $this->withAdminAccessToken()->getJson('api/v3/signups?orderBy=quantity,desc');
-        $decodedResponse = $response->decodeResponseJson();
-
-        $response->assertStatus(200);
-
-        // Assert results are returned in descending order.
-        $this->assertEquals(5, $decodedResponse['data'][0]['quantity']);
-        $this->assertEquals(4, $decodedResponse['data'][1]['quantity']);
-
-        // Order results by ascending quantity
-        $response = $this->withAdminAccessToken()->getJson('api/v3/signups?orderBy=quantity,asc');
-        $decodedResponse = $response->decodeResponseJson();
-
-        $response->assertStatus(200);
-
-        // Assert results are returned in ascending order.
-        $this->assertEquals(1, $decodedResponse['data'][0]['quantity']);
-        $this->assertEquals(2, $decodedResponse['data'][1]['quantity']);
-    }
-
```

Comment:
Is this functionality not something we do anymore?

Reply:
I don't think so (I believe it was added for Domino), and it's a little awkward to write out the test since this is calculated based on the posts. We could update this to sort by a different field if we want coverage for this behavior.

Reply:
Oh DOMINO blast from the past!

Reply:
So does this `orderBy` parameter not work anymore? I don't remember seeing anything about that in this PR so I'm having trouble wrapping my head around this

Reply:
It does! I had removed this test when I originally thought we weren't storing `quantity` on the post anymore, but should probably re-add it to keep the test coverage.

Reply:
Re-added this test in f0a485c.